docs/user/analytics.rst

```diff
@@ -1,5 +1,7 @@
-Traffic Analytics
-=================
+How To View Traffic Analytics
+=============================
+
+In this guide, you can learn to use Read the Docs' built-in traffic analytics for your documentation project and optionally add Google Analytics.
```

Comment:
Just wanted to mention that this guide also explains "how to _remove_ Google Analytics"

Reply:
Good idea! Fixed!